### PR TITLE
[TASK] Use guzzle for the smoke tests

### PR DESCRIPTION
**Pull request by sabbelasichon**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Refactoring and testability

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

### CODE REVIEW

File: src/Task/Test/Response.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace TYPO3\Surf\Task\Test;
```
Comment (t3easy) [inline src/Task/Test/Response.php]: Move model to model namespace
```suggestion
namespace TYPO3\Surf\Domain\Model;
```

File: src/Task/Test/Response.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+final class Response
```
Comment (t3easy) [inline src/Task/Test/Response.php]:
```suggestion
final class HttpResponse
```

**Comment by sabbelasichon** [comment]
@t3easy Can we merge this?

**Comment by kaystrobach** [comment]
looks good so far (initial look) any help needed to finalize it? Current beta has no smoketest at all :(

**Comment by sabbelasichon** [comment]
@kaystrobach Thanks. What do you mean, that the current betas has no smoketest at all? You mean, there is not test for the smoketest, that´s true. But the HttpTestTask already exists.

**Comment by kaystrobach** [comment]
Mhmm i searched for Smoketest and have not recognized that HttpTestTask

**Comment by kaystrobach** [comment]
just took a look into the 2beta-10 phar archive again ... the test folder is missing:

image

**Comment by kaystrobach** [comment]
Btw. the tag in the repo contains the directory.

https://github.com/TYPO3/Surf/tree/2.0.0-beta10/src/Task/Test

**Comment by Sebobo** [comment]
It's actually documented ;) https://github.com/TYPO3/Surf#smoke-testing

**Comment by Sebobo** [comment]
The issue is in the box.json. All folders called `Test` or similar are excluded

**Comment by sabbelasichon** [comment]
@Sebobo Fixed the box.json file

**Comment by kaystrobach** [comment]
How about renaming from Test to Verify or Proof? This would simply avoid the Term, inthink otherwise configuring the build can be tricky.

**Comment by sabbelasichon** [comment]
@kaystrobach Something for the future. I would like to keep it as it is for the moment.

**Comment by kaystrobach** [comment]
See removed `Test` but keept `Tests` ... thank you!

**Pull request merged**